xarray/backends/common.py

```diff
@@ -508,6 +554,17 @@ def guess_can_open(
 
         return False
 
+    def open_datatree(
+        self,
+        filename_or_obj: str | os.PathLike[Any] | BufferedIOBase | AbstractDataStore,
+        **kwargs: Any,
+    ) -> DataTree:
+        """
+        Backend open_datatree method used by Xarray in :py:func:`~xarray.open_datatree`.
+        """
+
+        raise NotImplementedError
```

Comment:
```suggestion
        raise NotImplementedError()
```

Reply:
I was pattern matching, should I update L545 too?